docs/source/api/v4/users_register.rst

```diff
@@ -31,7 +31,7 @@ Register a user and send registration email.
 Request Structure
 -----------------
 :email:    Email address of the new user The given email is validated (circuitously) by `GitHub user asaskevich's regular expression <https://github.com/asaskevich/govalidator/blob/9a090521c4893a35ca9a228628abf8ba93f63108/patterns.go#L7>`_ . Note that it can't actually distinguish a valid, deliverable, email address but merely ensure the email is in a commonly-found format.
-:role:     The integral, unique identifier of the highest permissions :term:`Role` which will be afforded to the new user. It restricts the allowed values to identifiers for :term:`Roles` with at most the same permissions level as the requesting user.
+:role:     The name of the :term:`Role` which will be afforded to the new user. It restricts the allowed values to identifiers for :term:`Roles` with at most the same permissions level as the requesting user.
```

Comment:
instead of `at most the same permissions level as the requesting user`, it should say `only Permissions the requesting user has.`

The model for Permissions has also changed from a numeric "level" to a collection of Permissions, so that wording is also legacy.

Reply:
Makes Sense